src/crud.py

```diff
@@ -308,3 +310,65 @@ def check_calls_per_month(db, contract_id):
         return True
     nb_operations_already_made = get_operations_by_contracts_per_month(db, contract_id)
     return max_calls >= len(nb_operations_already_made)
+
+
+def create_max_calls_per_sponsee_condition(
+    db: Session, condition: schemas.CreateMaxCallsPerSponseeCondition
+):
+    # If a condition still exists, do not create a new one
+    existing_condition = (
+        db.query(models.Condition)
+        .filter(models.Condition.sponsee_address == condition.sponsee_address)
+        .filter(models.Condition.vault_id == condition.vault_id)
+        .filter(models.Condition.current < models.Condition.max)
+        .one_or_none()
+    )
+    if existing_condition is not None:
+        raise ConditionAlreadyExists(
+            "A condition with maximum calls per sponsee is already existing and the maximum is not reached. Cannot create a new one."
+        )
+    db_condition = models.Condition(
+        **{
```

Comment:
I think nothing ^^' 
At first I wanted "deconstruct" `condition` parameter to put it into models.Condition with `**` but it didn't worked because a dict is needed to use `**`.
So I built a dict ... 🤣

Reply:
It's a leftover from my python tests xD